packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,257 @@
+import {
+  StackLayout,
+  Avatar,
+  Text,
+  Card,
+  Button,
+  Tooltip,
+  FlexLayout,
+} from "@salt-ds/core";
+import {
+  WomanIcon,
+  MessageIcon,
+  LocationIcon,
+  CallIcon,
+  ChatIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+} from "@salt-ds/icons";
+import { Meta } from "@storybook/react";
+import { useState } from "react";
+
+export default {
+  title: "Patterns/Contact Details",
+} as Meta;
+
+const basicContact = {
+  primary: "Alex Brailescu",
+  secondary: "Blackrock Advisors (UK) Limited",
+  metadata: [
+    {
+      label: "Role",
+      value: "Analyst",
+      icon: <WomanIcon />,
+    },
+    {
+      label: "Location",
+      value: "London, GBR",
+      icon: <LocationIcon />,
+    },
+    {
+      label: "Office",
+      value: "+44 2077 431102",
+      icon: <CallIcon />,
+    },
+    {
+      label: "Email",
+      value: "alex.brailescu@blackrock.com",
+      icon: <MessageIcon />,
+    },
+  ],
+};
+
+export const ContactDetails = ({
+  avatarSrc = "https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png",
+}) => {
+  return (
+    <StackLayout direction={"column"} gap={1}>
+      <Avatar
+        src={avatarSrc}
+        aria-label={basicContact.primary}
+        fallbackIcon={<WomanIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3">{basicContact.primary}</Text>
+        <Text styleAs="h4">{basicContact.secondary}</Text>
+        <StackLayout direction={"row"}>
+          <StackLayout gap={1} direction={"column"}>
+            {basicContact.metadata.map((metadata) => {
+              return (
+                <Text styleAs="label" key={metadata.label}>
+                  {metadata.label}
+                </Text>
+              );
+            })}
+          </StackLayout>
+          <StackLayout gap={1} direction={"column"}>
+            {basicContact.metadata.map((metadata) => {
+              return (
+                <Text styleAs="label" key={metadata.label}>
```

Comment:
same as above, also this text in uitk looks gray - would Text with variant="secondary" work here?